### PR TITLE
Rename survival x-axis label

### PR DESCRIPTION
**Pull request by dippindots**
Fix cBioPortal/cbioportal#7961

Describe changes proposed in this pull request:
- a Use the name of the _MONTHS attribute as the axis label

before:
image

after:
image

### CODE REVIEW

**Review by inodb — APPROVED** [review]

File: src/shared/lib/comparison/ComparisonStore.ts [hunk 1]
```diff
@@ -1864,6 +1868,44 @@ export default class ComparisonStore {
...
+            this.survivalClinicalAttributesPrefix,
+        ],
+        invoke: () => {
+            const survivalXAxisLabelGroupByPrefix = _.reduce(
+                this.survivalClinicalAttributesPrefix.result!,
+                (acc, prefix) => {
+                    const clinicalAttributeId = `${prefix}_MONTHS`;
+                    const clinicalAttributeMap = _.groupBy(
+                        this.activeStudiesClinicalAttributes.result,
+                        'clinicalAttributeId'
```
Comment (alisman) [inline src/shared/lib/comparison/ComparisonStore.ts:1883]: don't use string shortcuts b/c they aren't typesafe
(n)=>n.clinicalAttributeId

Comment (alisman) [reply src/shared/lib/comparison/ComparisonStore.ts:1883]: also, i don't think building this map is necessary.  just filter them by clinicalAttributeId=match.

File: src/shared/lib/comparison/ComparisonStore.ts [hunk 2]
```diff
@@ -1864,6 +1868,44 @@ export default class ComparisonStore {
...
+                        this.activeStudiesClinicalAttributes.result,
+                        attr => attr.clinicalAttributeId
+                    );
+                    const xLabels: string[] = [];
+                    if (
+                        clinicalAttributeMap &&
+                        clinicalAttributeMap[clinicalAttributeId] &&
+                        clinicalAttributeMap[clinicalAttributeId].length > 0
+                    ) {
+                        clinicalAttributeMap[clinicalAttributeId].forEach(
```
Comment (alisman) [inline src/shared/lib/comparison/ComparisonStore.ts:1891]: this is a mapping operation

Comment (alisman) [reply src/shared/lib/comparison/ComparisonStore.ts:1891]: should be xLabels = thing.map(...)

**Review by alisman — APPROVED** [review]

**Review by jjgao — APPROVED** [review]
Looks good from the product perspective.

**Pull request merged**